### PR TITLE
89: FUNCTION 'LXE_OBJ_DOKU_GET_XSTRING_RFC' does not exists

### LINKED ISSUE #896: check_89: FUNCTION 'LXE_OBJ_DOKU_GET_XSTRING_RFC' does not exists
CLASS zcl_aoc_check_89 claims that FUNCTION 'LXE_OBJ_DOKU_GET_XSTRING_RFC' exists.
However, this is not the case in my system running on S4HANA ON PREMISE 1909. 

```
    "function module already exists in 7.02 -> should work on every satellite system
    CALL FUNCTION 'LXE_OBJ_DOKU_GET_XSTRING_RFC'
      DESTINATION lv_destination
```

Well, as a workaround I simply can deactivate this check.

### PR DESCRIPTION
**Pull request by MiLiCLN**
Add Wrapper. 
Fix #896

### CODE REVIEW

File: src/utils/zcl_aoc_util_reg_atc_namespace.clas.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (larshp) [inline src/utils/zcl_aoc_util_reg_atc_namespace.clas.xml]: are you running the latest abapGit? and on a unicode system?

Comment (MiLiCLN) [reply src/utils/zcl_aoc_util_reg_atc_namespace.clas.xml]: abapGit 1.93.0 on a  unicode system

Comment (larshp) [reply src/utils/zcl_aoc_util_reg_atc_namespace.clas.xml]: can you double check? I get diffs after pulling, and this part of the XML is not normally changed

**Comment by larshp** [comment]
@MiLiCLN

**Comment by MiLiCLN** [comment]
> 
> 
> @MiLiCLN

@larshp 
What's the next step?

**Comment by larshp** [comment]
it looks like there is something wrong with the XML file, `src/utils/zcl_aoc_util_reg_atc_namespace.clas.xml`

can you double check that its the latest abapGit running?

**Comment by MiLiCLN** [comment]
> 
> 
> it looks like there is something wrong with the XML file, `src/utils/zcl_aoc_util_reg_atc_namespace.clas.xml`
> 
> can you double check that its the latest abapGit running?

With the latest abapGit version 1.100.0 I could not find any differences.
The only difference is probably the encoding (utf-8 with BOM)?

I cannot work directly from abapGit into my repo. In abapGit I have to take the workaround via "Package to Zip". In the zip-file all files are stored as utf-8 with BOM.
So far I have never paid attention to the encoding.

**Comment by larshp** [comment]
okay, something might be broken in that process

I've pulled to local and updated with https://github.com/larshp/abapOpenChecks/pull/940/commits/2367483f2708c73c72b1f09fa879d244f3a2ba79

merging

thanks 👍

**Pull request merged**